alma/lib/Utils/Settings.php

```diff
@@ -235,7 +236,8 @@ public static function getActiveAPIKey()
      */
     public static function getLiveKey()
     {
-        return self::get('ALMA_LIVE_API_KEY', '');
+        $encryption = new EncryptionHelper();
+        return $encryption->decrypt(self::get(ApiAdminFormBuilder::ALMA_LIVE_API_KEY, ''));
```

Comment:
self::get second parameter is an empty string to change parameter type? null is not good?

Reply:
@Francois-Gomis I think it's a same result, I replaced by null :)